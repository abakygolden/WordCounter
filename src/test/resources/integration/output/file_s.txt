S 12
SA 1
SABER 1
SABIDUR 1
SABLE 1
SAD 1
SAGITTIS 3
SAL 2
SALTEN 2
SALVAR 1
SAMMENH 1
SAPIEN 3
SAW 1
SE 7
SEA 1
SEAN 1
SECRET 1
SED 6
SEG 1
SEM 3
SEPAN 1
SER 4
SERVIR 1
SERVIRLE 1
SHE 2
SHIP 1
SI 5
SIEMPRE 2
SIG 2
SIN 5
SIR 1
SIT 5
SK 2
SKAL 2
SLENDER 1
SMALL 1
SNOW 1
SO 1
SOBERBIA 1
SODALES 2
SOFT 1
SOLLICITUDIN 1
SOM 7
SOMETIMES 2
SPEJLER 2
SPORTSBEGIVENHEDER 1
SPORTSLIG 1
SPORTSST 1
ST 3
STARS 1
STEDE 1
STERSTRAND 2
STICK 1
STIGNING 1
STOD 1
STRID 2
STRIKES 1
STRONG 1
SU 4
SUCEDE 1
SUCUMBIR 1
SUELE 1
SUERTE 1
SUNLIGHT 1
SUS 1
SUSCIPIT 1
SUSPENDISSE 1
SVENDE 1