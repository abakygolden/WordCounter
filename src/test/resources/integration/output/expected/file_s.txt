S 12
SAGUARO 4
SALAD 20
SALAK 4
SALMON 4
SANDWICH 12
SAPOTE 4
SATAY 4
SCAMPI 4
SHAWARMA 4
SHELLS 4
SHORTCAKE 4
SHRIMP 8
SLOPPY 4
SOUP 12
SOURSOP 4
SPRING 4
STARFRUIT 4
STEAK 4
STIR 8
STUFFED 12
SUB 4
SUGAR 4
SUSHI 4